cmd/config/config.go

```diff
@@ -55,13 +55,14 @@ type Routes struct {
 }
 
 type VendorRoutes struct {
-	GetAll              string `mapstructure:"get-all" validate:"required"`
-	UpdateDetail        string `mapstructure:"update-detail" validate:"required"`
-	GetById             string `mapstructure:"get-by-id" validate:"required"`
-	GetLocations        string `mapstructure:"get-locations" validate:"required"`
-	EmailBlast          string `mapstructure:"email-blast" validate:"required"`
-	AutomatedEmailBlast string `mapstructure:"automated-email-blast" validate:"required"`
-	Evaluation          string `mapstructure:"evaluation" validate:"required"`
+	GetAll                  string `mapstructure:"get-all" validate:"required"`
+	UpdateDetail            string `mapstructure:"update-detail" validate:"required"`
+	GetById                 string `mapstructure:"get-by-id" validate:"required"`
+	GetLocations            string `mapstructure:"get-locations" validate:"required"`
+	EmailBlast              string `mapstructure:"email-blast" validate:"required"`
+	AutomatedEmailBlast     string `mapstructure:"automated-email-blast" validate:"required"`
+	Evaluation              string `mapstructure:"evaluation" validate:"required"`
+	GetPopulatedEmailStatus string `mapstructure:"get-populated-email-status" validate:"required"`
```

Comment:
I wonder why is the name GetPopulatedEmailStatus? Not just GetEmailStatus? Does "Populated" describes anything?

Reply:
Yeah, that's right. I want to confirm, should we remove `GetEmailStatus` from the email package since it has already been moved to `GetPopulatedEmailStatus` in the vendor package to avoid confusion? If yes, then we'll also need to update the API in the frontend at the same time.

Reply:
Hemmm.... actually this is very interesting. Could you explain more about the circular import issue you mentioned? I rarely see that case ehehehe

Reply:
In our case, the circular import happened because the `vendor` package needed the `email` package to send emails, and the `email` package needed the `vendor` package to get or populate vendor information. Go immediately throws an `import cycle not allowed` error when this happens.

Go enforces this restriction to keep compilation fast. If circular dependencies were allowed, changing one part of the cycle would force Go to recompile the entire loop of dependencies to ensure everything works together, which would slow things down significantly, especially in larger projects. That’s why Go avoids them. it’s all about maintaining fast and efficient builds.

Reply:
Andrew have updated the /email frontend to use GET /vendor/email  API even this is not merged yet 😂. So I think frontend will not be affected if you want, however u should check if there is any other page that use GET /email-status.